### ISSUE TITLE
New panel type: for viewing URLs in sidebar

### ISSUE THREAD

**Issue by PeriodicOccupancy** [mbnuqw/sidebery#775]
Pretty much the only feature missing from this app imo to make it the ultimate all-in-one sidepar app would be the ability to pin URLs like panels and opening them would display the associated webpage where tabs or bookmarks are usually displayed. Would allow me to ditch my webpage sidebar addon.

I'm an Opera GX fan and the main thing stopping me from using Firefox as my daily driver instead of GX is the sidebar features of GX like the above mentioned one. Would also be nice to have a panel for history but thats less necessary.

**Comment by mbnuqw** [comment]
I experimented with this idea long time ago and iirc there were the next problems:
- Lots of modifications of the requests headers to allow the page to be showed inside iframe
  - Poor performance
- Questionable privacy/security

But generally speaking this is possible (an old screenshot):
![screenshot-2020-12-17 13:26:22](https://user-images.githubusercontent.com/6276694/192863452-c0895995-69a9-43ce-940d-25a25e98f54c.png)

I'll leave this issue open, maybe a bit later I'll revisit it, but for now it's a low priority feature.

**Comment by PeriodicOccupancy** [comment]
In that case, There is another issue, Sidebery can't switch panels (via hotkey) when Sidebery is not active in the sidebar, should I submit another issue for that? cause it prevents me from switching panels while I use a separate extension for viewing a webpage. the Simple Tab groups extension allows me to do that, but it's much less customizable and not as professional looking as Sidebery, and at that point, I'd rather stick with Opera GX

**Comment by mbnuqw** [comment]
@PeriodicOccupancy, yes, you can open a feature request for switching panels with inactive Sidebery sidebar.

**Comment by clushie** [comment]
> Questionable privacy/security

The current implementation of "preview in page" is also not secure? Or are the dom-insertions invisible for the website itself? (If I go to inspect page, and then hover the tabs I can see the element appearing inside the HTML `<body>` of the currently opened tab)

What was your concern on a Sidebar tab preview?
